lib/events/events.proto

```diff
@@ -80,6 +80,28 @@ message ConnectionMetadata {
 message KubernetesClusterMetadata {
     // KubernetesCluster is a kubernetes cluster name.
     string KubernetesCluster = 1 [ (gogoproto.jsontag) = "kubernetes_cluster,omitempty" ];
+    // KubernetesUsers is a list of kubernetes usernames for the user.
+    repeated string KubernetesUsers = 2 [ (gogoproto.jsontag) = "kubernetes_users,omitempty" ];
+    // KubernetesGroups is a list of kubernetes groups for the user.
+    repeated string KubernetesGroups = 3 [ (gogoproto.jsontag) = "kubernetes_groups,omitempty" ];
+}
+
+// KubernetesPodMetadata contains common metadata for kubernetes pod-related
+// events.
+message KubernetesPodMetadata {
+    // KubernetesPodName is the name of the pod.
+    string KubernetesPodName = 1 [ (gogoproto.jsontag) = "kubernetes_pod_name,omitempty" ];
```

Comment:
I think the reason for this redundancy is because this struct is embedded? Not sure why all these fields have to be top-level in the events (maybe so it's easier to handle on the UI) but that seems to be how it's done currently.

Reply:
Yeah, it's due to embedding and how events look when marshaled to JSON.
All these are embedded for JSON marshaling compatibility with the old events format (pre-4.4).
@klizhentas can correct me if I'm wrong.